README.md

```diff
@@ -219,6 +219,19 @@ org.graalvm.compiler.truffle.runtime.OptimizedCallTarget#executeRootNode
 org.graalvm.compiler.truffle.runtime.OptimizedCallTarget#profiledPERoot
 ```
 
+#### Describe a graph
+
+Describe the key features of a graph without rendering it.
+
+```
+% seafoam examples/fib-java.bgv.gz:1 describe
+21 nodes, branches, calls
```

Comment:
I think it would be very useful to report the number of branches, calls and loops and not just "there are some/none".